blockchains/optimism/assets/0x8700dAec35aF8Ff88c16BdF0418774CB3D7599B4/info.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Synthetix Network",
+    "website": "https://synthetix.io",
+    "description": "Synthetix is a derivatives liquidity protocol on Ethereum that enables the issuance and trading of synthetic assets.",
+    "explorer": "https://optimistic.etherscan.io/address/0x8700daec35af8ff88c16bdf0418774cb3d7599b4",
+    "research": "https://research.binance.com/en/projects/synthetix",
+    "type": "OPTIMISM-ERC20",
```

Comment:
```suggestion
    "type": "OPTIMISM",
```

Reply:
This one is tricky, which format we should follow here

Reply:
similar to polygon, optimism may have other token types